core/shared/src/main/scala/matryoshka/Recursive.scala

```diff
@@ -301,6 +256,185 @@ trait Recursive[T] extends Based[T] {
     (implicit R: Corecursive.Aux[R, Base], BF: Functor[Base])
       : R =
     cata[R](t)(R.embed(_))
+
+  def mapR[U, G[_]: Functor]
```

Comment:
This and below is pulled from `FunctorT` and `TraverseT` – it seemed prudent to not get rid of the operations _yet_ (#45 is tracking this), since we use them a bunch in Quasar and I’m just not sure what the right approach is.

There are also some operations moved from earlier in the type class – into the group of things that should be in `Birecursive` once that is actually usable.